### PR TITLE
[IMP] website_event: improve confirmation page

### PR DESCRIPTION
**Pull request by stefanorigano**
Closed in favor of #44843
-----

Improve confirmation page design. Reduce no-shows making
"add to calendar" links more visible.

TASK #2189545

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by stefanorigano** [comment]
Hello @qsm-odoo ,  apparently we have to many no-shows at our events, fp wants this to be merged asap in master and odoo.com.
@tbe-odoo  will take care of the merging... nothing crazy here but if you have time to have a look too it would be appreciated. Thanks!

**Comment by JKE-be** [comment]
Now that we cleaned openerp_website to remove event's custo and push it in standard, it seems a really bad idea to reapply custo from master to our odoo.com (v13). We just spend several days with @Cocographique to remove old 'temp' views apply on our prod that make the v13 buggy.

Probably (imho) that merge it in v13.2 and just enough and wait the next migration in a few weeks to get the imp is the best solution for odoo ecosystem, maintenance and next migrations.

File: addons/website_event/views/event_templates.xml [hunk 1]
```diff
@@ -675,60 +675,107 @@
...
-                        <a role="button" class="btn btn-primary" t-att-href="iCal_url">
-                            <i class="fa fa-fw fa-calendar"/> Add to iCal/Outlook
-                        </a>
-                        <a role="button" class="btn btn-primary" t-att-href="google_url" target='_blank'>
-                            <i class="fa fa-fw fa-calendar"/> Add to Google Calendar
-                        </a>
+        <t t-call="website.record_cover">
+            <t t-set="_record" t-value="event"/>
+            <t t-set="use_filters" t-value="True"/>
+            <t t-set="additionnal_classes" t-value="'o_record_has_cover cover_auto p-5 mb-n5'"/>
```
Comment (qsm-odoo) [inline addons/website_event/views/event_templates.xml]: Note: you can use `t-valuef` to not have to add the simple quotes.

**Comment by qsm-odoo** [comment]
\+ change the target of the PR to 13.2 and rebase your branch on it so we'll have that when we migrate to 13.2 as JKE explained :+1:

**Comment by stefanorigano** [comment]
> + change the target of the PR to 13.2 and rebase your branch on it so we'll have that when we migrate to 13.2 as JKE explained 👍

Closed in favor of #44843

**Comment by JKE-be** [comment]
@stefanorigano 

You can just change the based branch on the original pr instead to close and open a new one and push force the new rebased branch ;)

**Comment by stefanorigano** [comment]
> @stefanorigano
> 
> You can just change the based branch on the original pr instead to close and open a new one and push force the new rebased branch ;)

@JKE-be , I didn't thought about the possibility of change base branch, Indeed I wanted just to rename it at first... thanks for the tip ;)

**Pull request closed without merging**